File: AtCoderTraining/Easy/C_Lower.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;
int main()
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int n;
    cin >> n;

    vector<int> a(n);

    int ans = 0;

    for (auto &x : a)
        cin >> x;

    for (int i = 0; i < n; i++)
    {
        int cnt = 0;
        while (i + 1 < n && a[i + 1] <= a[i])
            i++, cnt++;
        ans = max(ans, cnt);
    }
    cout << ans << '\n';
}

File: AtCoderTraining/Easy/C_Lower.py
n = int(input())

a = list(map(int, input().split()))

ans = 0

for i in range(n):
    cnt = 0
    while i + 1 < n and a[i+1] <= a[i]:
        i+=1
        cnt += 1
    ans = max(ans, cnt)
    
print(ans)